wordpress-mobile/MediaEditor-iOS
# Fix crash in iOS 12

The `MediaEditor` was crashing in iOS 12 devices due to a difference in the API of `UINavigationController`.

### To test

1. Run the example app in an iOS 12 device or simulator
2. Check that the Media Editor appears

If you try the steps above in `develop`, you'll see that the app crashes.
